crates/primitives/src/lib.rs

```diff
@@ -16,30 +16,24 @@ pub mod specification;
 pub mod state;
 pub mod utilities;
 
-pub use bits::B160;
-pub use bits::B256;
+pub use alloy_primitives::{
+    self, address, b256, bytes, fixed_bytes, hex, hex_literal, ruint, uint, Address, Bytes,
+    FixedBytes, B256, U256,
+};
+
 pub use bitvec;
 pub use bytecode::*;
-pub use bytes;
-pub use bytes::Bytes;
 pub use constants::*;
 pub use env::*;
 pub use hashbrown::{hash_map, hash_set, HashMap, HashSet};
-pub use hex;
-pub use hex_literal;
 #[cfg(feature = "c-kzg")]
 pub use kzg::{EnvKzgSettings, KzgSettings};
 pub use log::Log;
 pub use precompile::*;
 pub use result::*;
-pub use ruint;
-pub use ruint::aliases::U256;
-pub use ruint::uint;
 pub use specification::*;
 pub use state::*;
 pub use utilities::*;
 
-/// Address type is last 20 bytes of hash of ethereum account
-pub type Address = B160;
 /// Hash, in Ethereum usually keccak256.
 pub type Hash = B256;
```

Comment:
Remove this one. I dont think it is used.

Reply:
https://github.com/bluealloy/revm/pull/724/commits/ee19b1edf8a52a7befe21d8bab79d7fbf6110930